contrib_bots/lib/github_issues.py

```diff
@@ -19,18 +19,13 @@ class IssueHandler(object):
 
     URL = 'https://api.github.com/repos/{}/{}/issues'
     CHARACTER_LIMIT = 70
-    CONFIG_FILE = '~/.github-issue-bot/github-issue-bot.conf'
+    CONFIG_FILE = '~/.github-auth.conf'
```

Comment:
CONFIG_FILE, REPO_NAME, and REPO_OWNER should be instance variables, not class variables.  Try to catch me or another mentor online in Zulip chat to discuss this further.